integration_tests/no_tensorflow_test.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+
+import keras_nlp
+
+
+class NoTensorflow(unittest.TestCase):
```

Comment:
Is this testing basic functions when TF is not installed? I'm curious to know how you make sure that TF is not installed for this test.

Reply:
It's in the actions.yml controlling github actions. When testing on jax and torch, as a last step we uninstall Tensorflow and run these tests.

This will give us some coverage is someone imports Tensorflow unconditionally. It should break this test.

Reply:
Got it! Thanks for explaining!